Sources/Persistence/CoreData/SiblingContextCoreDataStack.swift

```diff
@@ -30,10 +30,10 @@ public final class SiblingContextCoreDataStack: CoreDataStack {
                 shouldMigrateStoreAutomatically: Bool = true,
                 shouldInferMappingModelAutomatically: Bool = true,
                 storeLoadCompletionHandler: @escaping (Any, Error?) -> Void = { (store, error) in
-                    if let error = error {
-                        fatalError("💥: Failed to load persistent store \(store)! Error: \(error)")
-                    }
-                },
+        if let error = error {
```

Comment:
`let` should be on the next line?

Reply:
this is a default parameter value for the closure parameter, and it's being indented as if it were the main function's body code.

if this was done by swiftlint's auto correct, I think perhaps we should try to add an exception for it to avoid applying this indentation here.

Reply:
It's not directly related with SwiftLint. It was complaining about parameters not being vertically aligned. I had to put the closure on the same line as the parameter name and xcode indenter made the indentation like this. I will revert the change.